src/transformers/models/bert/modeling_bert.py

```diff
@@ -176,11 +177,21 @@ def __init__(self, config):
         # any TensorFlow checkpoint file
         self.LayerNorm = nn.LayerNorm(config.hidden_size, eps=config.layer_norm_eps)
         self.dropout = nn.Dropout(config.hidden_dropout_prob)
-
         # position_ids (1, len position emb) is contiguous in memory and exported when serialized
-        self.register_buffer("position_ids", torch.arange(config.max_position_embeddings).expand((1, -1)))
         self.position_embedding_type = getattr(config, "position_embedding_type", "absolute")
 
+        if version.parse(torch.__version__) > version.parse("1.6.0"):
+
+            self.register_buffer(
+                "position_ids", torch.arange(config.max_position_embeddings).expand((1, -1)), persistent=False
+            )
```

Comment:
I would leave the position IDs at they already are, it has been like that for a few versions already.

Reply:
reverted the changes on position_ids.